go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/jrick/logrotate v1.0.0
+	github.com/linden/localstorage v0.0.0-20231117043609-5d94f0a86609
```

Comment:
Perhaps we can just fold in the code into the project so we don't need to add a new extertnal dep? It could live in an `internal` package. 

I checked it out, and looks like a pretty minimal set of helper funcs.